bouture.js

```diff
@@ -63,6 +64,17 @@ function completeElement (tag, args) {
               return name.replace(/^once|on/, '').toLowerCase()
             }
 
+            function isAttribute (name) {
+              if (name.match(/^data/)) {
+                return name.match(/^data-([a-z]|[0-9]|-)*$/)
+              } else if (ATTRIBUTENAMESWITHTAGS.hasOwnProperty(name)) {
+                if (ATTRIBUTENAMESWITHTAGS[name] === '*' || ATTRIBUTENAMESWITHTAGS[name].indexOf(tag) !== -1) {
```

Comment:
`.indexOf(tag) !== -1` can be replaced by `.includes(tag)`. That said, it might be worth for these arrays to be turned into Set objects and use `.has`